### ISSUE TITLE
No maintainers?

### ISSUE THREAD

**Issue by hmaarrfk** [conda-forge/bisonpp-feedstock#5]
### Comment:

How can a recipe have 0 maintainers?

**Comment by xmnlab** [comment]
I am the original maintainer of this project. I created this package in the past because I needed that for a project, but I am not working on that anymore.

**Comment by xmnlab** [comment]
but if it is something critical, I can try to help or at lest I can try to bring someone to help here

**Comment by hmaarrfk** [comment]
i'm kinda confused who requested the package be migrated tbh. i would love to ping them to help ;)

**Comment by xmnlab** [comment]
you mean https://github.com/conda-forge/bisonpp-feedstock/pull/4 ? 
or you are talking about another thing?

I created this package because I needed it here: https://github.com/conda-forge/heavydb-ext-feedstock/blob/main/recipe/meta.yaml#L84 (long time ago, when the project name was omniscidb)

**Comment by hmaarrfk** [comment]
Yeah, somebody made a PR to https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/arch_rebuild.txt

adding some file that depends on this feedstock.

**Comment by xmnlab** [comment]
got it .. thanks for the explanation

**Comment by hmaarrfk** [comment]
@jakirkham you seem to have added omniscidb which dependds on bisonpp to https://github.com/conda-forge/conda-forge-pinning-feedstock/commit/ed3b7a798f9d5805db34e182deb9f00aa38e32d1

do you have any desire in maintaining this feedstock?

**Comment by hmaarrfk** [comment]
Well if somebody wants to maintain this correctly, I think a call to:
```
# Some combination of these should help regenerate the configure file
# for OSX, but I'm really not sure
# libtoolize --force
# aclocal
# autoheader
# autoconf --force
```

should help. I'm hacking through this for now.

**Comment by xmnlab** [comment]
thanks for sharing!
